packages/secure_storage/amplify_secure_storage_dart/lib/src/platforms/amplify_secure_storage_web.dart

```diff
@@ -56,14 +58,15 @@ class AmplifySecureStorageWeb extends AmplifySecureStorageInterface {
         database.createObjectStore(storeName);
       }
     };
-    return openRequest.future;
+    // setting _database instead of returning it is a work
+    // around for https://github.com/dart-lang/sdk/issues/48835
```

Comment:
```suggestion
    // TODO: update once https://github.com/dart-lang/sdk/issues/48835 
    // is resolved in a stable version. setting _database instead of returning 
    // it is a work around.
```